ts/features/itwallet/common/components/ItwUpcomingWalletBanner.tsx

```diff
@@ -1,14 +1,24 @@
 import React from "react";
-import { Banner } from "@pagopa/io-app-design-system";
-import { Linking } from "react-native";
+import {
+  Banner,
+  IOVisualCostants,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { Linking, View, StyleSheet } from "react-native";
 import I18n from "../../../../i18n";
 import { useIOSelector } from "../../../../store/hooks";
 import { trialStatusSelector } from "../../../trialSystem/store/reducers";
 import { itwTrialId } from "../../../../config";
 import { SubscriptionStateEnum } from "../../../../../definitions/trial_system/SubscriptionState";
 import { isItwEnabledSelector } from "../../../../store/reducers/backendStatus";
 
-export const ItwUpcomingWalletBanner = () => {
+type ItwUpcomingWalletBannerProps = {
+  ignoreMargins?: boolean;
```

Comment:
I think the banner should not take care of adding any margin or padding, that's something the consumer should take care of. What do you think? 

You are in fact sending `true` in the consumer screen because the margin is already present.

Reply:
Yes you are right. I'll fix